modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteTxAdapter.java

```diff
@@ -221,7 +221,10 @@
     private volatile TransactionState state = ACTIVE;
 
     /** Timed out flag. */
-    private volatile boolean timedOut;
+    protected volatile boolean timedOut;
```

Comment:
may be leave private and add protected getter

Reply:
and setter